src/packages/babel-preset-dev-toolkit/index.js

```diff
@@ -73,4 +67,6 @@ if (env === 'production') {
   ]);
 }
 
-module.exports = preset;
+module.exports = function(api, opts) {
```

Comment:
Maybe we can use the `arrow function` here, since we're already using babel ?

Reply:
yup, will change